### DISCUSSION TITLE
Servlet schism

### DISCUSSION THREAD

**Discussion by rossabaker** [http4s/http4s#6228] [Ideas]
* Jetty 9.4's [EOL is imminent](https://www.eclipse.org/lists/jetty-users/msg10029.html).  
* We are two major versions of the Servlet specification behind, and [v6 is due May 31](https://projects.eclipse.org/projects/ee4j.servlet/releases/6.0)

I have spun off the existing servlet code into the [http4s-servlet](https://github.com/http4s/http4s-servlet) module.  From there, we can make progress independent of core.  We have three basic options:

- Keep releasing 0.23.x from here and propagate changes there.  That's a lot of git-filter-repo folderol.
- Purge it from this repo and try to release it concurrently with core.  The g8 template requires this, and we don't have a great track record.  The problem is in triplicate, because Jetty and Tomcat are spinning off.
- Purge it from this repo and desynchronize servlet patches from core patches.  This requires educating our users.

**Comment by rossabaker** [comment]
If anyone wants to be on the servlet team, :heart: this comment.  @danicheg and @yuferpegom have made recent contributions, and @armanbilge is a build nerd.

**Reply by armanbilge** [reply]
Happy to help get the build off on the right foot, but can't afford to follow too closely after that :)

I'm very 👍 to purging from this repo. I think our two best hopes to maintaining the release cadence are:
1. recruiting (community) maintainers who care about these modules. I'd love to see a call-to-action like the one we did in TL.
2. running our own steward action after every 0.23.x release to propagate the update to satellites. then it's merge+release from GH UI, which isn't too bad.

**Reply by rossabaker** [reply]
We'll have to start with Tomcat and Jetty, and Jetty is going to be more complicated because it includes the client.
